rpcs3/Emu/RSX/Overlays/overlays.cpp

```diff
@@ -138,16 +138,18 @@ std::string utf16_to_ascii8(const std::u16string& utf16_string)
 
 std::u16string ascii8_to_utf16(const std::string& ascii_string)
 {
-	std::vector<char16_t> out;
+	std::u16string out;
 	out.reserve(ascii_string.length() + 1);
```

Comment:
With `out` changed to string, you don't need to reserve `+ 1` because you're not appending the null terminator no more.